cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -203,7 +203,7 @@
 adata_with_labels = anndata.AnnData(
     X=sparse.csr_matrix(X),
     obs=pd.concat([good_obs, obs_expected], axis=1),
-    var=pd.concat([good_var, var_expected], axis=1),
+    var=var_expected,
```

Comment:
Using `pd.concat` here corrupts the resulting dataframe which leads to a failed validation. `var_expected` is a superset of `good_var` so this is also fine.

Reply:
just curious -- why isnt that also an issue for obs?

Reply:
I think the issue is that duplicate columns are merged in a weird way (if you select the column, you get a `Dataframe` and not a `Series`). The two obs do not have duplicate columns.